frontend/src/__mocks__/mockRunKF.ts

```diff
@@ -0,0 +1,49 @@
+  },
+  // resource_references: [
```

Comment:
Looks like we can remove this section of resource references.

Reply:
its used in our source code for showing the pipeline runs

Reply:
This comment is referring to commented out code. Should it not be commented out?

Reply:
oh yeah i guess we can, it was not needed for my testing

Reply:
If you think there's a use for keeping it, we can, but might as well uncomment it then, unless that causes issues for some reason.

Reply:
it wont effect anything right now so i think its fine. these refs wont be used in v2 anyway